### PR TITLE
#5 expanded InputHasContent and TextHasString ctors

### PR DESCRIPTION
**Pull request by Vatavuk**
Added new ctors in `TextHasString ` and `InputHasContent ` that accept `Text` for #5

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#9](https://github.com/llorllale/cactoos-matchers/issues/9) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#9](https://github.com/llorllale/cactoos-matchers/issues/9) is assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/C63314D6Z/1371962685); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @llorllale[/z](https://www.0crat.com/u/llorllale) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer

File: src/test/java/org/cactoos/matchers/InputHasContentTest.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ *
+ * @author Vedran Vatavuk (123vgv@gmail.com)
+ * @version $Id$
+ * @since 1.1
+ * @checkstyle JavadocMethodCheck (500 lines)
+ */
+public final class InputHasContentTest {
+
+    @Test
+    public void matchesInputContent() throws Exception {
```
Comment (paulodamaso) [inline src/test/java/org/cactoos/matchers/InputHasContentTest.java]: @Vatavuk This test isn't working, it does not compares `String` and `InputOf` contents; this method should return `true` when using `new InputHasContent("Hello World!")`.

Comment (Vatavuk) [reply src/test/java/org/cactoos/matchers/InputHasContentTest.java]: @paulodamaso I'm not sure that I get what is the problem. I'm doing exactly as you described, comparing `Input` with `String`

**Comment by Vatavuk** [comment]
@paulodamaso fixed failure reason descriptions

File: src/test/java/org/cactoos/matchers/InputHasContentTest.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        final String text = "Hello World!";
+        final Input input = new InputOf(text);
+        final InputHasContent matcher = new InputHasContent(text);
+        MatcherAssert.assertThat(
+            "Matcher does not match equal values",
+            matcher.matchesSafely(input),
```
[the remark this answers is not in the document]
Comment (Vatavuk) [reply src/test/java/org/cactoos/matchers/InputHasContentTest.java]: @victornoel You are right. Thanks!

File: src/test/java/org/cactoos/matchers/TextHasStringTest.java [hunk 3]
```diff
@@ -34,6 +34,7 @@
 import org.hamcrest.StringDescription;
 import org.hamcrest.core.StringContains;
 import org.junit.Test;
+import org.llorllale.cactoos.matchers.TextHasString;
```
[the remark this answers is not in the document]
Comment (Vatavuk) [reply src/test/java/org/cactoos/matchers/TextHasStringTest.java]: @victornoel I will leave this for #12 to avoid multiple packages.

**Comment by Vatavuk** [comment]
@victornoel thanks for your comments.
@paulodamaso please take a look again

File: src/test/java/org/cactoos/matchers/TextHasStringTest.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (victornoel) [reply src/test/java/org/cactoos/matchers/TextHasStringTest.java]: @Vatavuk yes, you are right, let's not mix concerns

**Comment by paulodamaso** [comment]
@rultor merge please

**Comment by llorllale** [comment]
@rultor merge

**Comment by 0crat** [comment]
@elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova) please review this job completed by @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#9](https://github.com/llorllale/cactoos-matchers/issues/9) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by elenavolokhova** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova)

**Comment by yegor256** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/llorllale/cactoos-matchers/issues/9#issuecomment-392079009))

@yegor256 This is what I know about this job, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#9](https://github.com/llorllale/cactoos-matchers/issues/9) is not in scope
  * Job [footprint](https://www.0crat.com/footprint/C63314D6Z?q=%7Bjob%3A%27gh%3Allorllale%2Fcactoos-matchers%239%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by paulodamaso** [comment]
@llorllale I've still not received payment for this issue, please take a look.

**Comment by llorllale** [comment]
@0crat pay 20min @paulodamaso

**Comment by 0crat** [comment]
> @0crat pay 20min @paulodamaso  ([here](https://github.com/llorllale/cactoos-matchers/issues/9#issuecomment-395096090))

@llorllale The project is under-funded, you can't do it now, see [§49](http://www.zerocracy.com/policy.html#49)

**Comment by paulodamaso** [comment]
@llorllale nothing here yet, please take a look

**Comment by llorllale** [comment]
@0crat pay 20min @paulodamaso

**Comment by 0crat** [comment]
> @0crat pay 20min @paulodamaso ([here](https://github.com/llorllale/cactoos-matchers/issues/9#issuecomment-398388358))

@llorllale The project is under-funded, you can't do it now, see [§49](http://www.zerocracy.com/policy.html#49)

**Comment by paulodamaso** [comment]
@llorllale ping

**Comment by llorllale** [comment]
@0crat pay 20min @paulodamaso

**Comment by 0crat** [comment]
> @0crat pay 20min @paulodamaso ([here](https://github.com/llorllale/cactoos-matchers/issues/9#issuecomment-401423021))

@llorllale The user @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) is not a member of this project, see [§49](http://www.zerocracy.com/policy.html#49)

**Comment by llorllale** [comment]
@paulodamaso you were automatically removed from this sandbox project as soon as your rep surpassed 1024 (see http://www.zerocracy.com/policy.html#33). Please file a bug in the relevant ZC repo so that they can take a look to see what can be done in this situation. I agree that you should be paid based on work done, not on your current membership status for a project.

**Comment by paulodamaso** [comment]
@llorllale Thanks for the return, filed an issue.

**Pull request merged**